exchanges/okcoin/okcoin_types.go

```diff
@@ -160,13 +160,13 @@ type WebsocketOrderbookResponse struct {
 // WebsocketOrderBook holds orderbook data
 type WebsocketOrderBook struct {
 	Checksum  int64             `json:"checksum"`
-	Asks      [][2]okcoinNumber `json:"asks"` // [ Price, Quantity, depreciated, number of orders at the price ]
-	Bids      [][2]okcoinNumber `json:"bids"` // [ Price, Quantity, depreciated, number of orders at the price ]
+	Asks      [][2]types.Number `json:"asks"` // [ Price, Quantity, depreciated, number of orders at the price ]
```

Comment:
Despite not being related to this PR, when reviewing these changes we're expecting 4 items as per the comments but only pre-allocating 2. [The API also states 4 items for both REST and websocket](https://www.okcoin.com/api/v5/market/books?instId=BTC-USD). Let me know if you'd like to resolve it in this PR or we can add it as a task for a separate PR

Reply:
I'll pass on resolving it in this PR, please Bob 🙏 

@Beadko - Perhaps you'd like to take this one?

Reply:
sure